client-js/app/elements/labrad-nodes.html

```diff
@@ -180,9 +190,40 @@
     td.controls {
       width: 350px;
     }
+    #buttons {
+      height: 50px;
+    }
+    #buttons paper-icon-button {
+      margin: 5px;
+    }
+    .autostartFilterOn {
+      color: #000;
+    }
+    .autostartFilterOff {
+      color: #aaa;
+    }
   </style>
   <template>
-    <div>
+   <div id="container" class='fit'>
+   <paper-header-panel id="left-column">
+    <div class="paper-header" id="buttons">
+      <paper-icon-button
+        hidden$={{isAutostartFiltered}}
+        class='autostartFilterOff'
+        icon="stars"
+        id="star"
+        on-click="autostartFilter"
+        title="Show Only Autostart Servers"></paper-icon-button>
+      <paper-icon-button
```

Comment:
We should make an element that encapsulates this pattern of an icon toggle button that can change colors. Seems like that would be useful in a number of places. Not for this PR, obviously.